org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/publisher/TranslationImplementation.java

```diff
@@ -21,40 +21,56 @@
  */
 
 
-import java.util.Date;
-import java.util.Map;
-import java.util.Set;
-
+import org.apache.commons.lang3.StringUtils;
 import org.hl7.fhir.r5.model.Enumeration;
 import org.hl7.fhir.r5.model.PrimitiveType;
 import org.hl7.fhir.r5.utils.TranslatingUtilities.TranslationServices;
 
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+
 public class TranslationImplementation implements TranslationServices {
 
   // marker is used to see what content is not being translated (debugging)
   private static final String MARKER1 = ""; // "^";
   private static final String MARKER2 = ""; // "^^";
-  
+  private Map<String, String> keys = new HashMap<>();
```

Comment:
Preface all member fields with m, so this could be mKeys. Helps differentiate when debugging methods.